financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationViewModel.kt

```diff
@@ -172,13 +172,13 @@ internal class LinkStepUpVerificationViewModel @Inject constructor(
         // Mark networked account as selected.
         val activeInstitution = selectNetworkedAccounts(
             consumerSessionClientSecret = payload.consumerSessionClientSecret,
-            selectedAccountIds = setOf(selectedAccount.id),
+            selectedAccountIds = selectedAccounts.map { it.id }.toSet(),
         )
 
         // Updates manifest active institution after account networked.
         updateLocalManifest { it.copy(activeInstitution = activeInstitution.data.firstOrNull()) }
         // Updates cached accounts with the one selected.
-        updateCachedAccounts { listOf(selectedAccount) }
+        updateCachedAccounts { selectedAccounts }
```

Comment:
We’re not actually changing anything compared to what we retrieve from `getCachedAccounts()` a few lines above.

@carlosmuvi-stripe, can we remove this call?

Reply:
I think so!